src/Classes/DocumentUpdater.php

```diff
@@ -156,4 +158,19 @@ private function getDataPath(): string
     return $this->storePath . Store::dataDirectory;
   }
 
-}
\ No newline at end of file
+    /**
+     * Encode the array to a json string
+     * @param array $json_object
+     * @return string
+     */
+    public function encodeJson(array $json_object): string
```

Comment:
Type declarations are supported in PHP >= 7.4

Reply:
See https://github.com/rakibtg/SleekDB/pull/234#discussion_r878758462